package.json

```diff
@@ -29,51 +29,53 @@
     "test:all": "ember try:each"
   },
   "dependencies": {
+    "@floating-ui/dom": "^1.2.5",
     "babel-plugin-filter-imports": "^4.0.0",
     "babel6-plugin-strip-class-callcheck": "^6.0.0",
     "broccoli-funnel": "^3.0.5",
+    "ember-auto-import": "^2.6.1",
```

Comment:
Added this (and the `webpack` dependency) to avoid the [error](https://stackoverflow.com/questions/70615052/ember-addon-could-not-find-module-from-within-component) caused by the library import in the component